packages/web-components/fast-foundation/src/tooltip/tooltip.pw.spec.ts

```diff
@@ -158,12 +158,49 @@ test.describe("Tooltip", () => {
             await element.evaluate((node: FASTTooltip) => node.anchorElement?.id)
         ).toBe("anchor-default");
 
-        await element.evaluate(node => {
+        await element.evaluate((node: FASTTooltip) => {
             node.setAttribute("anchor", "new-anchor");
         });
 
         expect(
             await element.evaluate((node: FASTTooltip) => node.anchorElement?.id)
         ).toBe("new-anchor");
     });
+
+    test("should change anchor element when the `anchorElement` property changes", async () => {
+        await root.evaluate((node: HTMLElement) => {
+            node.innerHTML = /* html */ `
+            <fast-tooltip anchor="anchor-default">
+                Tooltip
+            </fast-tooltip>
+            <fast-button id="anchor-default">
+                Hover or focus me
+            </fast-button>
+            <fast-button id="anchor-new">
+                Hover or focus me
+            </fast-button>
+        `;
+        });
+
+        expect(
+            await element.evaluate((node: FASTTooltip) => node.anchorElement?.id)
+        ).toBe("anchor-default");
+
+        await element.evaluate((node: FASTTooltip) => {
+            const newAnchor = document.getElementById("anchor-new");
+            node.anchorElement = newAnchor;
+        });
```

Comment:
This approach seems duplicative. In this test, the anchor's ID is already known, so what benefit does setting the `anchorElement` provide over setting the `anchor` property directly?

Reply:
The point of the feature is to allow setting the anchor via an element instance, in the current implementation only an id works.  The main benefit of setting the anchor element directly is that it enables tooltips to be anchored to elements in shadow trees which can't be rereferred to via id.  As I write this I'm realizing I'd need to swap describedby/labelledby for description/label in that instance as well, rethinking...